content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,284 @@
+
+Remote configuration will be turned on by default on your new organization and you can create new API keys that are RC-enabled by default for use with your Agent. See the documentation for [Remote configuration][7] for more information.
+{{% /collapse-content %}}
+
+{{% collapse-content title=" 10. Update your Datadog Agent version and configuration" level="h5" %}}
+Update your Datadog Agents to version **7.54 or higher**. This version comes with a new configuration for Disaster Recovery which enables Datadog Agents to also send telemetry to the configured secondary Datadog site after DDR failover is activated. The Agent dual ships telemetry to support customers conducting periodic disaster recovery exercises/drills. 
+
+After updating your Agent, configure your Datadog Agent's `datadog.yaml` configuration file as shown in the example below and restart the Agent.
+
+```shell
+multi_region_failover:
+  enabled: true
+  failover_metrics: false
+  failover_logs: false
+  failover_traces: false
+  site: <DDR_SITE>  # For example "site: us5.datadoghq.com" for a US5 site
+  api_key: <DDR_SITE_API_KEY>
+```
+
+Setting the **enabled** field to `true` enables the Agent to ship Agent metadata to the secondary Datadog site so you can view Agents and your Infra Hosts in the secondary site. Note that while you can see your Agents and Infra Hosts at the secondary site, you will not receive telemetry until DDR failover is activated.
+
+During the preview, we recommend having `failover_metrics`, `failover_logs` and `failover_traces` set to **false** when in passive phases. 
+
+Your Datadog contact will work with you on scheduling dedicated time windows for failover testing to measure performance and Recovery Time Objective(RTO).
+{{% /collapse-content %}} <br>
+
+
+#### when you are ready to test the failover process
+{{% collapse-content title=" 11. Activate and test DDR failover" level="h5" %}}
+There are several methods that can be used for activating/testing the DDR failover. 
+
+**Agent in non-containerized environments**
+
+For Agent deployments in non-containerized environments, use the below Agent CLI commands:
+
+```shell
+agent config set multi_region_failover.failover_metrics true
+agent config set multi_region_failover.failover_logs true
+agent config set multi_region_failover.failover_traces true
+```
+
+**Agent in containerized environments**
+
+If you are running the Agent in a containerized environment like Kubernetes, the Agent command-line tool can still be used, but it needs to be invoked on the container running the Agent. 
+
+Below is an example of using `kubectl` to fail over metrics and logs for a Datadog Agent pod deployed via either the official Helm chart or Datadog Operator. The `<POD_NAME>` should be replaced with the name of the Agent pod:
```

Comment:
It looks like there are several options here that you can choose from, but that's a bit hard to see. You can use formatting to make that easier to understand! I would use bullet points for all of these and add an intro to say that you can pick one of three (I think?) methods.